Elliptic/cube_1x1x1_GroundWaterFlow/cube_1e0.prj

```diff
@@ -34,6 +34,7 @@
     <process_variables>
         <process_variable>
             <name>pressure</name>
+            <components>1</components>
```

Comment:
I think using `component_number` it is not so clear if it is a component index or a count of components.